README.md

```diff
@@ -19,3 +19,30 @@ Older releases may be found in the old ROS-Industrial [subversion repository][].
 [ROS wiki]: http://ros.org/wiki/abb
 [abb_experimental]: https://github.com/ros-industrial/abb_experimental
 [subversion repository]: https://code.google.com/p/swri-ros-pkg/source/browse
+
+## Build
+### Build from Binaries (Recommended)
+This build methods uses:
+ - [catkin tools](https://catkin-tools.readthedocs.io/en/latest/) for building.
+```
+mkdir -p abb_ws/src
+cd abb_ws/src
+git clone -b kinetic-devel https://github.com/ros-industrial/abb.git
+rosdep update
+rosdep install abb
+catkin build
+```
+
+### Build from Source (Developers Only)
+This build methods uses:
+ - [wstool](http://wiki.ros.org/wstool) for rosinstall.
+ - [catkin tools](https://catkin-tools.readthedocs.io/en/latest/) for building.
+```
+mkdir -p abb_ws/src
+cd abb_ws/src
+git clone -b kinetic-devel https://github.com/ros-industrial/abb.git
+wstool init .
+wstool merge abb/abb.rosinstall
+wstool update
+catkin build
+```
```

Comment:
And it's certainly implied, but if we're going to be this detailed about documenting build steps then perhaps we should add a `source devel/setup.bash` as well?